frontend/dispatcher.lua

```diff
@@ -286,50 +319,89 @@ local dispatcher_menu_order = {
     "embedded_fonts",
     "smooth_scaling",
     "nightmode_images",
+
+    "kopt_trim_page",
+    "kopt_page_margin",
+
+    "kopt_zoom_overlap_h",
+    "kopt_zoom_overlap_v",
+    "kopt_zoom_mode_type",
+    --"kopt_zoom_range_number", -- can'y figure out how this name text func works
+    "kopt_zoom_factor",
+    "kopt_zoom_mode_genus",
+    "kopt_zoom_direction",
+
+    "kopt_page_scroll",
+    "kopt_page_gap_height",
+    "kopt_full_screen",
+    "kopt_line_spacing",
+    "kopt_justification",
+
+    "kopt_font_size",
+    "kopt_font_fine_tune",
+    "kopt_word_spacing",
+    --"kopt_text_wrap", -- has multiple events
```

Comment:
May be we can have somewhere a single onReflowToggled that would itself send/broadcast the current 3 other events RedrawCurrentPage / RestoreZoomMode / InitScrollPageStates ?

Reply:
>Reflow: uses multiple events and dispatcher currently is only setup to handle one. Maybe i will add a handler in koptlistener, but not in this PR (doing so would simplify configdialog etc.)